CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## Changelog
 
+## Unreleased
+
+### Added
+
+* Warn when failing to start the preview server ([#130](https://github.com/swoosh/swoosh/pull/130))
+* Support for mail headers in Mailgun adapter ([#134](https://github.com/swoosh/swoosh/pull/134))
+* Allow adding attachments in Email.new ([#135](https://github.com/swoosh/swoosh/pull/135))
+
+### Misc
+
+* Mention async implementation in [README](https://github.com/swoosh/swoosh#async-emails)
```

Comment:
I wouldn't put that here, this changelog is meant to be about changes in the software itself imo. What do you think?